apps/condo/domains/common/components/ResidentActions/ResidentActions.tsx

```diff
@@ -122,6 +123,7 @@ export const ResidentActions: React.FC<IResidentActionsProps> = (props) => {
     ), [setIsSearchByPhoneModalVisible, isAssignedVisibilityType, setDropdownVisible])
 
     const trigger: DropDownProps['trigger'] = useMemo(() => isMobile ? ['click'] : ['hover'], [isMobile])
+    const iconSize: IconProps['size'] = minified ? 'medium' : 'small'
```

Comment:
`minified` is not `small`?

Reply:
Yep, on `minified` version there's no Label on button, that's why icon can be a bit larger